src/components/LineChart/LineChart.styles.js

```diff
@@ -7,18 +7,14 @@ import isIE from 'isIE';
 
 export const Container: ComponentType<*> = (() => {
   return styled.div`
-    grid-column: span 3;
+    grid-column: 2/-2;
     display: flex;
     flex-direction: column;
 
     & > span {
       font-weight: normal;
     }
 
-    @media only screen and (max-width: 768px) {
```

Comment:
I think we should keep these media queries. On a mobile the charts should span full width.

Reply:
I made them full widths at first on desktop too, but then it look a bit too big, so I changed them. Sure thing, I'll put them back.